### DISCUSSION TITLE
Welcome to laravel-vue3-spa-starter Discussions!

### DISCUSSION THREAD

**Discussion by irabbi360** [irabbi360/laravel-vue3-spa-starter#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edinzujodev** [comment]
I've tried spinning the app on lando, however the user cant login. This is the error msg: "message:"Unauthenticated."".
It works perfectly when I try artisan serve. 
Maybe because lando is working on different port? Do you have any idea how can I solve this issue? I would appreciate any help!

**Reply by irabbi360** [reply]
For Unauthenticated errors, the main reason is the app URL. To the .env set the app URL of your application URL then restart the server.
